packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -605,6 +607,11 @@ - (void)_initAppWithUrl:(NSURL *)appUrl bundleUrl:(NSURL *)bundleUrl manifest:(E
 
 - (BOOL)isAppRunning
 {
+  if([_appBridge isProxy]){
+    RCTHost *reactHost =  [self.delegate rctAppDelegate].rootViewFactory.reactHost;
```

Comment:
the `[self.delegate rctAppDelegate]` is weak reference so it could be nil, we should handle this case